MapleServer2/Types/Player.cs

```diff
@@ -39,6 +39,8 @@ public class Player
         public int PrestigeLevel = 100;
         public long PrestigeExperience;
         public int TitleId;
+        public List<int> AvaliableTitles = new List<int> { 0 };
+        public List<short> AvaliableInsignias = new List<short> { 0 };
```

Comment:
Available is spelled wrong here, but  I also think it should just be `Titles` and `Insignias`, the Available doesn't really provide extra information since you can only have one title/insignia equipped therefore the plural version necessitates it means available